src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calc = {
```

Comment:
it's up to you, but you could return the object right away without declaring a variable
```suggestion
  return {
```

Reply:
Anyway, well done!

Reply:
Yep. Good advice from Vova